src/dotnet-ildasm/Disassembler.cs

```diff
@@ -136,7 +138,8 @@ private void WriteTypeSignature(TypeDefinition type)
             //TODO: Signature to use IL types #2
             _outputWriter.Write($" {type.FullName}");
             //TODO: External Types should always be preceded by their assembly names #6
-            _outputWriter.Write($" extends {type.BaseType.FullName}");
+            if(!type.IsInterface)
```

Comment:
"Interfaces inherit from zero or more base interfaces; therefore, this property returns null if the Type object represents an interface."

But we could use  GetInterfaces or  FindInterfaces, to find the base ones.